test/cypress/grid/integration/options/virtualization-threshold.cy.js

```diff
@@ -0,0 +1,42 @@
+describe('Grid rows virtualizaion threshold', () => {
+    before(() => {
+        cy.visit('grid-pro/cypress/virtualization-threshold');
+    });
+
+    it('Should properly set rows virtualizaion based on row count.', () => {
+        cy.window().its('Grid').then(({ grids }) => {
+            const grid = grids[0];
+
+            // Virtualization should be active since the row count exceeds the
+            // default threshold.
+            expect(grid.options.rendering.rows.virtualization).to.be.true;
```

Comment:
Can we change it to something easier to read in case it fails?
```js
            assert.isTrue(
                grid.options.rendering.rows.virtualization,
                'Virtualization should be active since the row count exceeds the default threshold.'
            );
```

Reply:
Good point, changed.